src/GeneralStateTestsFiller/stReturnDataTest/tooLongReturnDataCopyFiller.yml

```diff
@@ -0,0 +1,264 @@
+tooLongReturnDataCopy:
+  _info:
+    comment: Ori Pomerantz   qbzzt1@gmail.com
+
+# When a contract ends (RETURN or REVERT), it returns a buffer. That buffer
+# is available either directly (written to memory specified in the CALL opcode)
+# or through RETURNDATACOPY. However, if you attempt in RETURNDATACOPY to read
+# beyond the buffer size the contract reverts. This test verifies this behavior.
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentNumber: 1
+    currentTimestamp: 1000
+    currentGasLimit: 0x10000000000000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+    currentBaseFee: 10
+
+  pre:
+    # Return the requested amount of data
+    000000000000000000000000000000000000C0DE:
+      balance: 0
+      code: |
+        :yul {
+           // How many bytes to return
+           let byteCount := calldataload(0)
```

Comment:
According to Robert Martin:

bytesToReturn 

Then comment is not needed
Or even bytesCountToReturn